client-react/src/pages/app/log-stream/LogStream.types.ts

```diff
@@ -25,11 +25,11 @@ export const newLine = '\n';
 export const maxLogEntries = 1000;
 
 export class LogRegex {
-  public static readonly infoLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+) (\[Info|INFO)/;
-  public static readonly infoLogUTC: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)[Z\s] (\[Info|INFO)/;
-  public static readonly errorLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+) (\[Error|ERROR)/;
-  public static readonly errorLogUTC: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)[Z\s] (\[Error|ERROR)/;
-  public static readonly warningLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+) (\[Warning|WARNING)/;
-  public static readonly warningLogUTC: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)[Z\s] (\[Warning|WARNING)/;
+  public static readonly infoLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+) (\[Info|INFO|\[INFO)/;
+  public static readonly infoLogUTC: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)(\Z|Z:) (\[Info|INFO|\[INFO)/;
+  public static readonly errorLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+) (\[Error|ERROR|\[ERROR)/;
+  public static readonly errorLogUTC: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)(\Z|Z:) (\[Error|ERROR|\[ERROR)/;
+  public static readonly warningLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+) (\[Warning|WARNING\[WARNING)/;
+  public static readonly warningLogUTC: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)(\Z|Z:) (\[Warning|WARNING\[WARNING)/;
```

Comment:
Add missing `|` delimiters.
```suggestion
  public static readonly warningLog: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+) (\[Warning|WARNING|\[WARNING)/;
  public static readonly warningLogUTC: RegExp = /^(\d{4}-\d{2}-\d{2})[T\s](\d{2}:\d{2}:\d{2}\.\d+)(\Z|Z:) (\[Warning|WARNING|\[WARNING)/;
```